CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Change Log
```

Comment:
Got template from http://keepachangelog.com/
But I don't mind towrite in one word